CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+# dbt_quickbooks v0.13.0
+## 🚨 Breaking Changes 🚨:
+[PR #124](https://github.com/fivetran/dbt_quickbooks/pull/124) includes the following updates:
+
+## Bug Fixes
+- Updates the [int_quickbooks__invoice_join](models/intermediate/int_quickbooks__invoice_join.sql) and downstream [quickbooks__ap_ar_enhanced](dbt_quickbooks/models/quickbooks__ap_ar_enhanced.sql) models to include the `using_payments` config. Previously, this model would fail if the `payment` or the `payment_line` source tables did not exist.
+- Requires [quickbooks__ap_ar_enhanced](dbt_quickbooks/models/quickbooks__ap_ar_enhanced.sql) to have `using_payments` enabled in order to run, otherwise the model is limited in its use.
```

Comment:
These two bullets are essentially saying the same thing. Is there a way to consolidate or streamline?

Reply:
consolidated!